### PR TITLE
Update wiki link in README.MD

### PR DESCRIPTION
**Pull request by squnit**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,8 +18,8 @@ If there is no data available, crawl from the website, then analyse data from lo
 > **First Rule: BE GOOD AND TRY TO FOLLOW A WEBSITE’S CRAWLING POLICIES. Don't crawl the website to often!**
 
 ## wiki
-- https://puppylpg.github.io/python/csgo/2019/12/01/python-crawler-buff.html;
-- https://puppylpg.github.io/python/csgo/2019/12/07/python-crawler-buff-optimaze.html;
+- https://puppylpg.github.io/2019/12/01/python-crawler-buff/
```
Comment (puppylpg) [inline README.md:21]: https://puppylpg.github.io/2019/12/02/python-crawler-buff/

**Review by puppylpg — COMMENTED** [review]
哈哈哈，之前网站换了个模板，地址变了，当时懒得改后来就忘了。

有个url写错了，我comment了，再改一下吧

File: README.md [hunk 2]
```diff
@@ -18,8 +18,8 @@ If there is no data available, crawl from the website, then analyse data from lo
...
+- https://puppylpg.github.io/2019/12/07/python-crawler-buff-optimaze/
```
Comment (squnit) [inline README.md:22]: Here, optimize is spelled incorrectly as optimaze. I am not sure if you want to update this in your original blog as well.

File: README.md (hunk 1 above)
Comment (squnit) [reply README.md:21]: I will fix this in the following commit.

File: README.md [hunk 3]
```diff
@@ -18,7 +18,7 @@ If there is no data available, crawl from the website, then analyse data from lo
 > **First Rule: BE GOOD AND TRY TO FOLLOW A WEBSITE’S CRAWLING POLICIES. Don't crawl the website to often!**
 
 ## wiki
-- https://puppylpg.github.io/2019/12/01/python-crawler-buff/
+- https://puppylpg.github.io/2019/12/02/python-crawler-buff/
 - https://puppylpg.github.io/2019/12/07/python-crawler-buff-optimaze/
```
Comment (squnit) [inline README.md:22]: Here, optimize is spelled incorrectly as optimaze. I am not sure if you want to update this in your original blog as well.

**Pull request merged**